token/methods_test.go

```diff
@@ -4,8 +4,9 @@ import (
 	"encoding/json"
 	"testing"
 
-	ma "github.com/anoideaopen/foundation/mock"
 	"github.com/stretchr/testify/require"
+
+	ma "github.com/anoideaopen/foundation/mock"
```

Comment:
It is customary for us to format the import section from 2 sections. 
Please delete all empty lines in the import section and execute the goimport command.